### PR TITLE
okta stepup post

### PR DESCRIPTION
**Pull request by edunham**
TODO: 
 - [x] edunham editorial review + fixups, grammarly
 - [ ] Colton editorial review
 - [ ] Additional tech review?
 - [ ] advocacy team consultation on title (edunham drives this)
 - [ ] Double check that outbound github repo link, make sure it's in dedicated repo w/ README

### CODE REVIEW

**Comment by edunham** [comment]
@coltonbeach this is ready for editorial review :)

@alisaduncan thoughts on a good title for this one?

File: _source/_posts/2023-03-22-stepup-okta.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image: blog/step-up-okta/stepup.jpg
+type: awareness
+---
+
+# Step-up Authentication with Okta Workforce Identity Cloud
```
Comment (alisaduncan) [inline _source/_posts/2023-03-22-stepup-okta.md]: The title is h1, so we should denote each section of the post with h2

File: _source/_posts/2023-03-22-stepup-okta.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+Step-up authentication in an application is a pattern of allowing access to non-critical resources using a basic level of authentication, and requiring additional authentication for critical resources. This offers a balance between frictionless user experience and security in an application.
+
+There are several methodologies to provide the functionality as discussed [here](/blog/2023/03/08/step-up-auth). 
+
+In this article we will explore few ways [Okta Workforce Identity Cloud (WIC)](https://www.okta.com/workforce-identity/) can support step-up authentication in an application.
+
+
+## Modern Application using OpenID Connect (OIDC)
```
Comment (alisaduncan) [inline _source/_posts/2023-03-22-stepup-okta.md]: The h2 and h3 tags are used for SEO so each section should enforce the keywords and terms we want to apply towards SEO

File: _source/_posts/2023-03-22-stepup-okta.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+* The following diagram describes how WIC processes `acr_values` during `/authorize` call:
+
+
+{% img blog/step-up-okta/1-step-up-authentication-acr-flow.jpg alt:"ACR flow diagram" width:"800" %}
+
+* Currently WIC supports a pre-defined [list](https://developer.okta.com/docs/guides/step-up-authentication/main/#predefined-parameter-values) of `acr_values`. These values are mapped to assurance levels supported in WIC [authentication policies](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/policies/about-app-sign-on-policies.htm).
+
+* The non-Okta-specific defined values such as `phr` and `phrh` are taken from [this](https://openid.net/specs/openid-connect-eap-acr-values-1_0.html#OpenID.PAPE) OIDC spec. 
+
+* [This guide](https://developer.okta.com/docs/guides/step-up-authentication/main/) explains the capability in more detail.
```
Comment (alisaduncan) [inline _source/_posts/2023-03-22-stepup-okta.md]: I recommend listing the name of the document here.

File: _source/_posts/2023-03-22-stepup-okta.md [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+    "sub": "00u9vme99nxudvxZA0h7",
+    "updated_at": 1490198843,
+    "zoneinfo": "America/Los_Angeles"
+}
+``` 
+``` 
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
```
Comment (alisaduncan) [inline _source/_posts/2023-03-22-stepup-okta.md]: I recommend changes all instances of SSWS tokens to use OAuth for Okta

File: _source/_posts/2023-03-22-stepup-okta.md [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+Okta supports the above transactional MFA scenario using [OIDC Client-Initiated Backchannel Authentication Flow (CIBA)](https://developer.okta.com/docs/guides/configure-ciba/main/)
+
+We will look at CIBA based scenario in more detail in a later article.
+
+
+## In summary
+
+Whether you application is using modern protocols such as OpenID Connect or built from legacy technologies, Okta's Workforce Identity Cloud can help you prompt users to step-up their authentication only when necessary for accessing sensitive resources. 
+
+Use your [Okta Developer Account](https://developer.okta.com/signup/) to test different step-up authentication flows and determine which is best for your application!
```
Comment (alisaduncan) [inline _source/_posts/2023-03-22-stepup-okta.md]: We should add CTA here to follow OktaDev on YouTube, Twitter, and perhaps some other posts that may be of interest.

**Review by alisaduncan — DISMISSED** [review]
Changes requested.

Also more changes may be requested on Monday

**Comment by shawnrecinto-okta** [comment]
Just checking in on the status of this @edunham it looks like there are some outstanding change approvals needed

**Review by isemona — APPROVED** [review]

**Pull request merged**